### PR TITLE
Cypress remove vgc and types

### PR DESCRIPTION
**Pull request by SophieDePaula**
Remove `vgc` and `types` packages

### CODE REVIEW

File: js/packages/eyes-cypress/src/browser/socketCommands.js [hunk 1]
```diff
@@ -78,19 +78,6 @@ function socketCommands(socket, refer) {
       return arg;
     }
   }
-
-  function getType(value) {
```
Comment (SophieDePaula) [inline js/packages/eyes-cypress/src/browser/socketCommands.js:82]: I don't use this and forgot to remove it

File: js/packages/eyes-cypress/src/plugin/errorDigest.js [hunk 2]
```diff
@@ -69,9 +69,9 @@ function errorDigest({passed, failed, diffs, logger, isInteractive}) {
 }
 
 function stringifyTestResults(testResults) {
-  const hostDisplaySize = testResults.getHostDisplaySize();
-  const viewport = hostDisplaySize ? `[${hostDisplaySize}]` : '';
-  const testName = `${testResults.getName()} ${viewport}`;
+  const hostDisplaySize = Object.values(testResults.hostDisplaySize);
+  const viewport = hostDisplaySize ? `[${hostDisplaySize[0]}x${hostDisplaySize[1]}]` : '';
```
Comment (kyryloonufriiev) [inline js/packages/eyes-cypress/src/plugin/errorDigest.js:73]: Why do you need Object.values? Isn't it just width and height?

File: js/packages/eyes-cypress/src/plugin/hooks.js [hunk 3]
```diff
@@ -21,7 +20,7 @@ function makeGlobalRunHooks({closeManager, closeBatches, closeUniversalServer})
         for (const summary of summaries) {
           const testResults = summary.results.map(({testResults}) => testResults);
           for (const result of testResults) {
-            testResultsArr.push(new TestResults(result));
+            testResultsArr.push(result);
           }
```
Comment (kyryloonufriiev) [inline js/packages/eyes-cypress/src/plugin/hooks.js:24]: Looks like this loop has no point

**Review by kyryloonufriiev — CHANGES_REQUESTED** [review]
Looks good to me. Feel free to merge once you address my small comments.

**Pull request merged**